ui/components/ui/box/box.d.ts

```diff
@@ -290,7 +290,7 @@ export interface BoxProps extends React.HTMLAttributes<HTMLElement> {
    * Use TEXT_ALIGN const from '../../../helpers/constants/design-system';
    * Accepts responsive props in the form of an array.
    */
-  textAlign?: BoxTextAlign | BoxTextAlignArray;
+  textAlign?: BoxTextAlign | BoxTextAlignArray | TextAlign;
```

Comment:
Left a comment in another of your PRs addressing this

Reply:
Got it!